source/_integrations/switcher_kis.markdown

```diff
@@ -23,7 +23,7 @@ Supported devices:
 - Switcher Power Plug
 - Switcher Touch (from firmware 1.51)
 - Switcher V2 (Espressif chipset - from firmware 3.21)
-- Switcher V2 (Qualcomm chipset - from firmware 72.32)
+- Switcher V2 (Qualcomm chipset - from firmware 72.32) Note: make sure port 20002 with udp is open.
```

Comment:
Does this need intructions?

Reply:
I had to open a different switcher library in order to see that it uses a non-standard port my firewall blocked.
I wasted an hour or so after reading thr homesssistant instructions because it was not noted.

Reply:
Oh wait, so this is no opening a port on the device, but on firewall you have created on  the Home Assistant end?

Reply:
Port 20002 applies to all devices, not only to `Switcher V2`, this note is confusing, it also make a standard user think he needs to do something, while the standard configuration not using IOT vlan does not need any action from the user.

However, I don't mind adding this as the last part of the documentation, see an example for [webOS TV notes](https://www.home-assistant.io/integrations/webostv/#notes)

@frenck do you think we should add it? I know that in general we don't add instructions for advanced installations in integrations docs.

Reply:
I think its fine to add a generic note at the bottom of the page, but in the current setting it is even more confusing IMHO.

Reply:
webOS TV notes looks like a better way to add it. And thanks for confirming that the port is the same for all devices. shall i close this pr and open a new one with the note at the bottom?

Reply:
@guysoft you can copy the example from [webOS TV notes](https://www.home-assistant.io/integrations/webostv/#notes) and if you are already there also add that **port 9957 TCP** also needs to be open (this probably works for you since you don't have a firewall from HA side to the device, but some others might have firewall in both directions.

Reply:
> webOS TV notes looks like a better way to add it. And thanks for confirming that the port is the same for all devices. shall i close this pr and open a new one with the note at the bottom?

both options are good

Reply:
Done and done, see https://github.com/home-assistant/home-assistant.io/pull/22086